<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="22" column="0" alignment="Qt::AlignRight">
       <widget class="QLineEdit" name="height_resolution"/>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>검색</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="18" column="0">
       <widget class="QComboBox" name="camchangeBox">
        <item>
         <property name="text">
          <string>0번</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1번</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="19" column="0">
       <widget class="QPushButton" name="camchangeButton_2">
        <property name="text">
         <string>전환</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="maximumSize">
         <size>
          <width>336</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>등록</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="14" column="0" rowspan="2">
       <widget class="QLabel" name="vidLabel">
        <property name="minimumSize">
         <size>
          <width>360</width>
          <height>240</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="20" column="0" alignment="Qt::AlignRight">
       <widget class="QLineEdit" name="width_resolution">
        <property name="maximumSize">
         <size>
          <width>659</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="23" column="0">
       <widget class="QPushButton" name="resolutionButton">
        <property name="text">
         <string>변환</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QOpenGLWidget" name="openGLWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
